### PR TITLE
Home work 2

### PR DESCRIPTION
**Pull request by YanaKopyshchyk1**

### CODE REVIEW

File: hw2/client.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const querystring = require('querystring');
```
[the remark this answers is not in the document]
Comment (YanaKopyshchyk1) [reply hw2/client.js]: Ok, I will rework it :)

File: hw2/index.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const server = http.createServer(async (req, res) => {
 
     if (pathname === '/anything' && method === 'GET') {
         try {
-            const response = await httpClient.get('https://jsonplaceholder.typicode.com/todos/1')
+            const response = await httpClient.get('https://jsonplaceholder1.typicode.com/todos/1')
```
Comment (viktornord) [inline hw2/index.js]: seems to be a mistake

File: hw2/index.js [hunk 3]
```diff
@@ -47,7 +48,8 @@ const server = http.createServer(async (req, res) => {
             res.end(JSON.stringify(responseData))
         })
     } else {
-        res.end()
+        res.statusCode = 404
```
Comment (viktornord) [inline hw2/index.js]: looks better. FYI consider this constant https://nodejs.org/api/http.html#http_http_status_codes for future. You can take advantage of it next time

File: hw2/client.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (viktornord) [reply hw2/client.js]: It is already `http.request` no need to rework. For future try to do it from scratch on your own

**Pull request merged**